guides/source/debugging_rails_applications.md

```diff
@@ -888,7 +888,7 @@ do that with `local_variables`.
 
 ### Settings
 
-* `config.web_console.whitelisted_ips`: Authorized list of IPv4 or IPv6
+* `config.web_console.allowlisted_ips`: Authorized list of IPv4 or IPv6
```

Comment:
These options are passed directly on to web console, and `whitelisted_ips` didn't change over there so this needs to stay the same.